k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml

```diff
@@ -137,14 +140,9 @@
     "sha256:63d5e04551ec8b5aae83b6f35938ca5ddc50a88d85492d9731810c31591fa4c9": [ "v4.0.2" ]
 - name: nfsplugin
   dmap:
-    "sha256:ec71d1797d1af17563bc5d19dfcf0229ca0f631d4a245f64ac358f7499946ee4": [ "v3.0.0" ]
```

Comment:
what's the impact of removing these items?

Reply:
keeping it as it is .